### ISSUE TITLE
wait for all the route sources before signalling the first load done event

### ISSUE THREAD

**Issue by aryszka** [zalando/skipper#839]
the problem serving 404s on startup was reported here: #835 
it was addressed in this PR: #837 

The referenced PR doesn't consider waiting for the first batch of route configuration from all the possible sources (DataClients). The functionality could be extended so that it signals only after all the datasources returned the first batch of routes.

**Comment by AlexanderYastrebov** [comment]
Duplicate of #1709
